DataConnectors/NXLogLinuxAudit.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "NXLogLinuxAudit",
+    "title": "NXLog LinuxAudit",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog [LinuxAudit](https://nxlog.co/documentation/nxlog-user-guide/im_linuxaudit.html) data connector supports custom audit rules and collects logs without auditd or any other user-space software. IP addresses and group/user ids are resolved to their respective names making [Linux audit](https://nxlog.co/documentation/nxlog-user-guide/linux-audit.html) logs more intelligible to security analysts. This REST API connector can efficiently export Linux security events to Azure Sentinel in real-time.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "LinuxAudit_CL",
+            "baseQuery": "LinuxAudit_CL"

```

Comment:
As per the json provided in sample data, custom table name is NXLogLinuxAudit_CL. Can you please either change the file name here https://github.com/Azure/Azure-Sentinel/pull/1280/files#diff-ac84e6e8f76926156578d14386bf7e10086ea3ba9901c2c60bbf00b86307f3a8 to LinuxAudit_CL or please use NXLogLinuxAudit_CL in all queries (sample queries, graph queries, datatypes and connectivitycriteria) ?

Reply:
This was resolved in the latest commit [08dec290](https://github.com/Azure/Azure-Sentinel/commit/08dec290af28748496d5ed66779cf2fef6718dab).

Reply:
Could not find the updates of the commit 08dec290 in this branch. Can you please merge the changes to this branch? 
![NxLogCommit](https://user-images.githubusercontent.com/73506651/100577074-45a63080-3294-11eb-9621-98727b94174b.PNG)